### ISSUE TITLE
Narrator can't announce the full information when go through the RadioButton control by using Up/Down/Left/Right arrow

### ISSUE THREAD

**Issue by Zheng-Li01** [dotnet/winforms#5235]
* .NET Core Version: 
.NET 6.0.100-preview.7.21363.2

* Have you experienced this same bug with .NET Framework?: 
 No

**Problem description:**
Narrator can't announce the full information when go through the RadioButton control by using Up/Down/Left/Right arrow, which is regression issue, can't reproduce on .NET Core 3.1.

**Expected behavior:**
Narrator can announce the full information when go through the RadioButton control by using Up/Down/Left/Right arrow.
![image](https://user-images.githubusercontent.com/38325459/125552046-d732abb0-2972-449a-8021-66464cbbb1fa.png)
![Issue_Core_31](https://user-images.githubusercontent.com/38325459/125552054-7ec3c198-a159-452a-acd5-85eecdfb159f.gif)

**Actual behavior:**
Narrator can't announce the full information when go through the RadioButton control by using Up/Down/Left/Right arrow.
![image](https://user-images.githubusercontent.com/38325459/125552038-f0f9075b-1bd0-4bcb-a29f-0598ceee2526.png)
![Issue_Core](https://user-images.githubusercontent.com/38325459/125552070-a9f45091-5262-4bd2-a2db-76a235ac601c.gif)

**Minimal repro:**
1. Open the attached application.
[Application_Issue2.zip](https://github.com/dotnet/winforms/files/6813130/Application_Issue2.zip)
2. Build and run it.
3.  Go through the RadioButton control by using Up/Down/Left/Right arrow.
4. Open the Narrator tool, and observe its behavior.

**Comment by merriemcgaw** [comment]
@Zheng-Li01 are you saying the expectation is that Narrator should announce the radio button you're leaving and the one you're going into with the arrow keys? What happens if you are using tab keys to move between without changing values, does that work OK?

**Comment by Zheng-Li01** [comment]
> @Zheng-Li01 are you saying the expectation is that Narrator should announce the radio button you're leaving and the one you're going into with the arrow keys?
 
Yes, due to the value was changed on this behavior.

> What happens if you are using tab keys to move between without changing values, does that work OK?

Can't use the tab keys to move between without changing values, however can do it by using Up/Down arrows with **On-Screen Keyboard**, please see below GIF.
![Update_5235](https://user-images.githubusercontent.com/38325459/126252022-911fa35f-bf4a-4630-8cba-1c28fb72aeef.gif)
